src/data/generated.ts

```diff
@@ -1326,6 +1334,16 @@ export type LoggedInUser = {
   networkId?: Maybe<Scalars['Int']>;
 };
 
+export type NetworkContractsInput = {
+  version?: Maybe<Scalars['String']>;
+  feeInverse?: Maybe<Scalars['String']>;
```

Comment:
I have small talk with @area about feeInverse. As fee inverse is coming straight from the contract and we cannot change it from FE. So the question is, if we passing it as a props because we want save it in cache? 🤔  or there is different reason behind that?

Reply:
Nope, you are correct. It just because we save it in the local resolver's cache.

This is just for us to be able to set it locally.